cobalt/site/docs/development/setup-android.md

```diff
@@ -33,6 +33,11 @@ return and complete the following steps.
 
     If prompted, read and accept the license agreement.
 
+1.  Install JDK 17. google it for how to. Verify java version is 17 and above.
```

Comment:
Do we care which variety of the JDK they download? Our internal instructions say to use OpenJDK but our CI uses the Oracle version.

Reply:
No, both works. The gradle plugin only care about the version.